src/basic/missing.h

```diff
@@ -557,7 +557,7 @@ struct btrfs_ioctl_quota_ctl_args {
 #define IFLA_INET6_ADDR_GEN_MODE 8
 #define __IFLA_INET6_MAX 9
 
-#define IFLA_INET6_MAX	(__IFLA_INET6_MAX - 1)
+#define IFLA_INET6_MAX  (__IFLA_INET6_MAX - 1)
```

Comment:
just fixed up an evil tab, replaced it by space... didn't want to make this a seperate "untabify" commit...